app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +18,9 @@
         tools:targetApi="31">
         <activity
             android:name=".MainActivity"
+            android:configChanges="orientation"
```

Comment:
What is the intention behind this change?

Reply:
Hi, On the Home screen there is the option to switch between Light/Dark mode.
https://developer.android.com/guide/topics/resources/runtime-changes

Reply:
Hi, I think I may have been experimenting with this for configChanges related to Light/Dark mode and with orientation. I may have failed to remove it as dev progressed. Removed.